slices/redis-server.yaml

```diff
@@ -0,0 +1,29 @@
+package: redis-server
+
+essential:
+  - redis-server_copyright
+
+slices:
+  bins:
+    essential:
+      - redis-server_config
+      - redis-tools_bins
+    contents:
+      /usr/bin/redis-server:
+
+  config:
+    contents:
+      /etc/default/redis-server:
```

Comment:
Should I remove the `/etc/default/redis-server` file?
I think it's only used with the init.d script I removed?

Reply:
Good point. I think so. Not just because it's used by the init.d script, but also because there isn't a `source` or dot operator that could be used to load that config before running `redis-server`

Reply:
Don.e